salt/modules/opkg.py

```diff
@@ -550,9 +550,16 @@ def install(name=None,
     if not cmds:
         return {}
 
+    updated_feeds = {}
     if refreshdb:
-        refresh_db()
-
+        updated_feeds = refresh_db()
+    failed_to_update_feeds = []
+    for feed in updated_feeds:
```

Comment:
I would like to see here a list comprehension. Something like this:
failed = [url for (url, success) in updated_feeds if not success]